nixos/doc/manual/from_md/release-notes/rl-2211.section.xml

```diff
@@ -2,8 +2,8 @@
   <title>Release 22.11 (“Raccoon”, 2022.11/30)</title>
   <para>
     The NixOS release team is happy to announce a new version of NixOS
-    22.11. NixOS is both a Linux distribution, and a set of packages
-    usable on other Linux systems and macOS.
+    22.11. NixOS is a Linux distribution, whose set of packages can be
+    used on other Linux systems and macOS.
```

Comment:
What about making Nixpkgs explicit?
```suggestion
    22.11. NixOS is a Linux distribution based on Nixpkgs.
    Nixpkgs is a package collection which can be also used on other Linux systems and macOS.
```

Reply:
This is what I was leaning towards, I think it may be better.